src/Recipe.js

```diff
@@ -6,8 +6,8 @@ class Recipe {
     this.image = recipe.image;
     this.ingredients = recipe.ingredients;
     this.instructions = recipe.instructions;
-    this.favorited = false;
-    this.toCook = false;
+    this.favorited = recipe.favorited ? recipe.favorited : false;
```

Comment:
This isn't crucial, but I've seen a lot of variables use the `is/has` standard for booleans. Think `isFavorited` or `willCook`. It reads as a boolean to other devs more readily, in my opinion.

Reply:
Thanks, good note on the tensing, fixed that up and utilized that handy VScode search feature @Tristan-Holtz ;)